recipes/edyn/all/conanfile.py

```diff
@@ -24,83 +29,72 @@ class EdynConan(ConanFile):
         "fPIC": True,
         "floating_type": "float",
     }
-    generators = "cmake", "cmake_find_package_multi"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
-    def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
-
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
-    def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
 
     def requirements(self):
-        self.requires("entt/3.9.0")
+        if Version(self.version) < "1.2.0":
+            self.requires("entt/3.9.0")
+        else:
+            self.requires("entt/3.10.1")
 
     @property
     def _compiler_required(self):
         return {
             "gcc": "9.3", # GCC 9.3 started supporting attributes in constructor arguments
+            "clang": "8",
+            "apple-clang": "10",
         }
 
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
```

Comment:
Let's just block it 🙈 I can not repo with this version either

```suggestion
    def validate(self):
        if self.options.shared and self.settings.compiler == "gcc" and Version(self.settings.compiler.version) == "11" and self.settings.build_type = "Release":
                    raise ConanInvalidConfiguration(f"{self.ref} does not build in C3i as shared with gcc-11 in release. Please submit a PR to fix this")
        if self.settings.compiler.get_safe("cppstd"):
```

Reply:
If you can see the rest of the builds, can you confirm if the other versions (1.0.0 and 1.2.0) are passing with that config? And also if libstdc++ is fine with everything else as such? If so I'll add self.version and libcxx conditions there too, so we can block as little as possible

Reply:
You can check the logs in https://github.com/conan-io/conan-center-index/pull/13562#issuecomment-1284854656

There's an "All Logs" hyperlink for the failed configuration. 😉 you should see them urself. Otherwise I just edit the URL https://c3i.jfrog.io/c3i/misc/summary.html?json=https://c3i.jfrog.io/c3i/misc/logs/pr/13562/11-linux-gcc/edyn/1.0.0//summary.json says eveything passed but I forgot to check this exact config before I closed the tab

Last time 1.2.0 failed. I assume (since those new header in the test package havent changed) the problem will be in all 3 versions.

Reply:
I see, very helpful! 
I'll leave it on all versions of Edyn for now then to be on the safe side.